modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/DropwizardServerGenerator.scala

```diff
@@ -463,7 +455,7 @@ object DropwizardServerGenerator {
                   )
                 )
 
-                val futureResponseType = completionStageType(responseType)
+                val futureResponseType = completionStageType(responseType.clone().setTypeArguments(new WildcardType))
```

Comment:
So now to implement this, I have to have the `<?>` in my method types?

```diff
-    CompletionStage<AddPetResponse> addPet(final Pet body);
+    CompletionStage<AddPetResponse<?>> addPet(final Pet body);
```

Reply:
Unfortunately yes.  It's a little visually ugly, but in practical terms isn't a big deal.  Alternatively we could do:
```
<T> CompletionStage<AddPetResponse<T>> addPet(final Pet body);
```
... but not sure that's a big improvement.

Reply:
What's the motivation behind this change? The previous behaviour matches the Scala strategy, since `isInstanceOf` and sealed trait matching are roughly equivalent. The only thing this doesn't do is exhaustivity checking. Could we implement these members in terms of `enum` to get exclusivity matching, without exposing this `<?>` stuff to the user?

Reply:
The main motivation is to make the generated code more compact (and nicer to read), and also to avoid some copy/paste between different server generator implementations.  Java of course doesn't have the exhaustivity checking, so any bugs around emitting all the if/else branches would need specific tests.

It's also nicer from a testing perspective; you can write tests that don't need to enumerate every single response type when building or testing mocks.  (This is a benefit our users would get, not just a benefit for guardrail itself.)